modules/apps/headless/headless-builder/headless-builder-impl/src/main/resources/com/liferay/headless/builder/internal/batch/00-list-type-definition.batch-engine-data.json

```diff
@@ -12,6 +12,60 @@
 		"taskItemDelegateName": "DEFAULT"
 	},
 	"items": [
+		{
+			"defaultLanguageId": "en_US",
+			"externalReferenceCode": "HTTP_METHOD_PICKLIST",
```

Comment:
I think this should be sorted alphabetically by externalReferenceCode

Reply:
I sorted it the same way it was sorted in the other file but it probably makes sense now that is separate. I'm on it.

Reply:
I think in the first file, the scope was different. Here we are just handling type definitions, so if there it no dependency between them, it must be sorted somehow